js/views/pauseScreen.js

```diff
@@ -318,7 +318,7 @@ class PauseScreen extends React.Component {
         </div>
 
         <button
-          id="oo-pause-button"
+          ref={btn => this.pauseButton = btn}
```

Comment:
Logic inside templates is a kind of antipattern

Reply:
it would be better to save function for 'ref' now.